adapters/onetag/onetagtest/supplemental/required-publisher-id.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Please consider adding a publisher id test where there's only one unique publisher id mixed with other imps which define an empty publisher id.

Reply:
I added this test.